src/wp-admin/js/media-upload.js

```diff
@@ -55,6 +88,11 @@ var tb_position;
 				tbWindow.css({'top': 20 + adminbar_height + 'px', 'margin-top': '0'});
 		}
 
+		/**
+		 * @summary Removes any width and height parameters from the href of any anchor with the ThickBox class and recalculates the new height and width values.
+		 *
+		 * @since 2.6.0
+		 */
 		return $('a.thickbox').each( function() {
```

Comment:
Please move this to the correct function. Probably the code was updated so the documentation block is now in the wrong position.